### ISSUE TITLE
Ambiguous language defining Audio Description in Understanding 1.2.5

### ISSUE THREAD

**Issue by gmjellin** [w3c/wcag#2441]
A common misinterpretation of 1.2.5 that I have noticed among accessibility professionals is that a separate Audio Description Track is required to narrate visual content, when in fact, the Audio Description may be included in the main (and only) soundtrack.

If my understanding is correct, "Audio Description" (AD) is narration that describes relevant information in the video content (information about actions, characters, scene changes, on-screen text, and other visual content.) AD may be part of the main soundtrack, or a separate track. 

Some of the language on Understanding implies that the AD must be a separate track (in addition to the main audio track). For example, the first line of the Note (green box) in the "Intent" section states, "For 1.2.3, 1.2.5, and 1.2.7, if all of the information in the video track is already provided in the audio track, **no audio description is necessary**". To me, the phrasing "no audio description is necessary" seems to be conflating AD with a separate audio track. 

Another example, from the first line of the "audio description" definition in the "Key Terms" section states, "narration added to the soundtrack to describe important visual details that cannot be understood from the main soundtrack alone". This language is ambiguous to me in that it states "...added to the soundtrack..." followed by, "main soundtrack alone". This implies that there are 2 soundtracks (soundtrack and main sound track).

I think it would be very beneficial to very clearly define Audio Description as the narration that may be included in the primary soundtrack or in a separate soundtrack.

**Comment by mitchellevan** [comment]
#2441 is a valid issue. The Understanding pages or Techniques pages need to spell out known valid techniques for newcomers to the topic.

It's well established by precedent that a single audio track can suffice for 1.2.3 and 1.2.5. Audio description in a single (not separate) audio track is often called [open audio description](https://digital.gov/2014/06/30/508-accessible-videos-how-to-make-audio-descriptions/).

We should further clarify two methods of open audio description. The first method is to have a narrator just for audio description, interleaving snippets of audio description into the audio track. Example: Two characters have a conversation in a TV show. Sam: "What's that thing doing here?" Audio description narrator (not a character): "Sam points to a pink unicorn." Pat: "Oh, that's just Freddy, he's harmless."

The second method is to have a speaker who voices both the primary narrative and the audio description. Example: A trainer narrates a prerecorded training video. One of the training slides depicts a three-step process: "Plan," "Create," "Deliver." The trainer says: "Our process is to plan each project, create the content, and deliver it. It's important not to skip the planning step!" The trainer has "added" audio description in the sense that she made sure to convey the visual content through her narrative.

Method 1 is typical for both open audio description and closed audio description. I have only seen method 2 used for open audio description, not for closed audio description.

**Comment by mitchellevan** [comment]
Scoping this issue:

* The present issue (2441) is to clarify whether open audio description is a sufficient technique for 1.2.3 and 1.2.5.
* Not to be confused with #1768, which seeks to clarify the distinction between 1.2.5 and 1.2.7.

I mention this because the [concurrent WAI-IG thread](https://lists.w3.org/Archives/Public/w3c-wai-ig/2022AprJun/0076.html) touches on both of these issues questions.

**Comment by bruce-usab** [comment]
I will volunteer to suggest an edit to understanding to help clarify the WCAG use of the term Audio Description.  It is a little different than the Video Description or how the term is used in the audio-video industry (namely, that AD is always on its own track).

> if all of the information in the video track is already provided in the audio track, no audio description is necessary

I agree that is confusing.  Really, it is that no _additional_ narration is necessary to meet the (WCAG) definition for Audio Description.  The note in the [definition](https://www.w3.org/TR/WCAG22/#dfn-audio-descriptions) does better (emphasis added):
> Where all of the video information is already provided in existing audio, no **additional** audio description is necessary.
  



Or maybe even: 
> if all of the information in the video track is already provided in the audio track, then audio description is already being provided

That is is awkward!  Please be encouraged to suggest something better.

**Comment by mitchellevan** [comment]
How about this:

* In the "Intent" section, Note 1, add the word "additional"
* In the "Examples of Success Criterion 1.2.5" section, replace the current single example with three examples:
   * A secondary audio description track ("closed audio description")
   * Audio description in the primary audio track ("open audio description")
   * Video information already provided in the primary audio track ("integrated description").

I thought it would be hard to find an example of open audio description, but in fact the current WGBH example would suffice! Videos are broken in the original link. I found the same or similar video here: https://www.pbslearningmedia.org/resource/tdc02.sci.life.evo.chenpdr/teaching-evolution-case-studies-bonnie-chen/

**Comment by silverli** [comment]
WAI does a wonderful job of delineating the differences for types of audio description in their page about [description of visual information](https://www.w3.org/WAI/media/av/description/#introduction). I would recommend including the same or similar language in the "Intent" section of 1.2.5:

> [Audio] description explains visual information needed to understand the content. (For example, “Pat opens a small box, looks at a diamond engagement ring, and cries”.) Depending on the type of video and media player, [audio] description can be:
> 
> * integrated into the main script and recording
> * narration added to the main audio track and provided in an alternative version of the video
> * provided in a text file or as a separate audio track

I also like the idea of defining open and closed audio descriptions as @mitchellevan stated.

However, for the utmost clarity, and due to the AV/arts industry norm of audio description generally meaning a separate track, changing the 1.2.5 normative statement as well to include the note (which I have changed a bit) "if all of the information in the video track is already integrated or provided in the audio track, no additional audio track is necessary" would probably make things clearest.

**Comment by bruce-usab** [comment]
@silverli &mdash; just to be totally candid, I am not inclined to suggest an edit to the SC text unless there is something wrong/incorrect/terrible.

> if all of the information in the video track is already integrated or provided in the audio track, no *additional audio track* is necessary

That seems to imply that providing AD narration in a separate *additive* audio track is a possibility.  (And I believe that has been experimented with, but is not implemented anywhere.)  How about: 

> if all of the information in the video track is already integrated into and provided by the audio track, no additional audio description is necessary

I am also thinking about a stand-alone sentence that the norm in the AV/arts industry is for AD to be distinct separate track, and is similar to how a movie on a DVD might provide the feature of a choice of languages.

**Comment by gmjellin** [comment]
@mitchellevan love this, particularly the terminology of open/closed desc. 

> In the "Examples of Success Criterion 1.2.5" section, replace the current single example with three examples:
> - A secondary audio description track ("closed audio description")
> - Audio description in the primary audio track ("open audio description")
> - Video information already provided in the primary audio track ("integrated description").

@silverli, I think that language from WAI would be a perfect addition to the 1.2.5. It articulates the main options for including AD very well and it is consistent with WAI, which is always nice.

> [Audio] description explains visual information needed to understand the content. (For example, “Pat opens a small box, looks at a diamond engagement ring, and cries”.) Depending on the type of video and media player, [audio] description can be:
> - integrated into the main script and recording
> - narration added to the main audio track and provided in an alternative version of the video
> - provided in a text file or as a separate audio track

@bruce-usab I agree that the term AD is often used to mean a distinct separate track, and it even seems to in the 1.2.5 Intent/Key Terms/Examples (though not entirely explicitly). @mitchellevan That is why I do agree with @silverli (though I think the recommendation of changing the SC text has been removed) that the use of the term Audio description in the SC text

> **Audio description** is provided for all prerecorded video content in synchronized media

is problematic because it is AD is widely understood to mean a distinct separate track.  I think many who read the SC with that understanding of the term conclude that the SC does indeed require a separate track. I realize changing the SC text is a big deal and I think it is acceptable to solve the ambiguity with the other clarifying language in the Understanding doc.

I really appreciate everyone's effort and ideas here.

**Comment by awkawk** [comment]
I don't agree with the suggested change - it impacts the definition of audio description and will create further confusion. When a video contains all of the information about the visual aspect that is needed to understand it in the primary program audio, no AD is needed. That's why the definition for audio description has the note: Where all of the [video](https://www.w3.org/TR/WCAG21/#dfn-video) information is already provided in existing [audio](https://www.w3.org/TR/WCAG21/#dfn-audio), no additional audio description is necessary.

There is also a note in the understanding document that reads: For 1.2.3, 1.2.5, and 1.2.7, if all of the information in the video track is already provided in the audio track, no audio description is necessary.

What is the actual problem that people are trying to solve?

**Comment by silverli** [comment]
@bruce-usab As long as audio description is defined clearly somewhere, I agree, I think the normative success criteria doesn't have to change, however, changing it would aid the most people because I know that many many people out there do think audio description means a separate track, and do not read the fine print of the "Understanding" sections (but that's another problem, I suppose). I also like your edits to the note:

>> If all of the information in the video track is already integrated into and provided by the audio track, no additional audio description is necessary

And like your idea here:
>> I am also thinking about a stand-alone sentence that the norm in the AV/arts industry is for AD to be distinct separate track, and is similar to how a movie on a DVD might provide the feature of a choice of languages.

@awkawk I agree with @gmjellin about the ambiguity of the definition of AD in 1.2.5, so this is my write-up/restatement of how I understand the issue we're discussing here:

WCAG 1.2.5 makes the **normative** statement that “Audio description is provided for all prerecorded video content in synchronized media,” however, it also says right below in the **non-normative** notes section that “if all of the information in the video track is already provided in the audio track, no audio description is necessary.” So it's saying both that audio description must be provided, but also that audio description is unnecessary “if all of the information in the video track is already provided in the audio track.”  But the normative statement is the one which holds the most weight as the required rule. The definition of audio description becomes ambiguous and the two statements seemingly conflict (saying AD should be provided in the normative statement, but then in the notes that it can also be unnecessary). It then becomes a bit of a mental scramble to figure out what “audio description” means, or how it could perhaps mean two things in this context: both integrated audio which contains sufficient description of the video, and also audio as a separate track.

**Comment by gmjellin** [comment]
@awkawk, The crux of the problem as I see it is that many people (A11y pros and others), interpret this SC to mean that a separate AD track is required for all synchronized media. I am currently a company that sells a product that has thousands of synchronized media (videos). They are running up against potential customers who are claiming that the product is not compliant with 1.2.5 and can't purchase the product until every video has a separate AD track. I've run into similar version of this problem at least half a dozen times.

If the SC had language that didn't include AD, but instead stated something like, "Narration describing important visual details is provided for all prerecorded video content in synchronized media" (imperfect perhaps, but bare with me), then the Understanding document would largely be fine as is with some tweaks.

Because the SC uses the term AD, which to some is defined as _narration describing important visual details that may be included in the primary (integrated) soundtrack or a separate track in addition to the primary track_, but to others is defined as _narration describing important visual details in a separate track in addition to the primary track_, there is frequent confusion as to whether a separate track is required or not. 

I agree that if you spend enough time reading and rereading the Understand doc (at least for some of us with perhaps marginal comprehension) it can be understood as it is intended to mean that the narration is included in the primary track or a separate track. However, I can promise you from personal experience, way too many people are not able to grasp that and it is causing a lot of confusion.

**Comment by awkawk** [comment]
@gmjellin When reading WCAG, there is only one definition of Audio Description and that is the normative one: 
narration added to the soundtrack to describe important visual details that cannot be understood from the main soundtrack alone

If I was selling a product with video in it, I would produce an ACR that reads as follows for 1.2.5:
Supports. [Product] includes many videos which have been carefully produced to ensure that all visual information necessary to understand the content is described in the primary program audio in order to eliminate the need for a separate audio description track. Following the guidance of the WCAG standard in the definition of audio description which reads "Where all of the video information is already provided in existing audio, no additional audio description is necessary", this Success Criterion is satisfied within this product.

I don't see how any customer could take issue with that, unless they are saying that your audio is missing important information so additional audio description is necessary.

**Comment by gmjellin** [comment]
@awkawk I was not aware that the Key Terms section is classified as normative so thank you for helping me understand that. And, I think that the example commentary for an ACR you included would help clarify and inform the reader. 

I still firmly believe, as do others, that when taken in it's entirety, the SC and the Understanding doc still contain a lot of ambiguity and the reality is many people misinterpret this SC. That is really my concern. I think continuing to strive for clearer less ambiguous language is important to make sure WCAG is more widely understood and utilized. 

I think for some who deeply understand WCAG and have been involved with it for many years (as I believe applies to you) it may seem much simpler and clearer. But imagine for a moment that you not as experienced and attempting to interpret this information.

I'll leave it at that. This has been a fruitful exercise for me personally in clarifying my own understanding.

**Comment by bruce-usab** [comment]
Thanks @awkawk for that very concrete ACR example!

@gmjellin &mdash; people familiar with theatrical quality AD frequently assume SC 1.2.5 requires a separate track (if not separate version).  Absent a change to WCAG 2.x itself, where is the best place for that clarification?

**Comment by awkawk** [comment]
@gmjellin I'm sure that there are many people who have spent less time reading and understanding (struggling to understand) WCAG than I, @bruce-usab, and others on the group have. I do appreciate the perspective you are bringing from your customers and see where there are challenges reading the document. 

The definitions are normative, for sure, which is why I'm concerned with putting a statement into the understanding document that says that "[audio] description can be integrated into the main script and recording".

I do agree that 1.2.5+definition leaves some room for confusion, but don't want to add to that. If I were writing this SC today, I would suggest that Audio description should be a technique to meet an SC that would be phrased as something like "All visual information presented in Synchronized media must be available in audio". That might be something that could be in a WCAG 3.0.

**Comment by gmjellin** [comment]
@awkawk, 

> All visual information presented in Synchronized media must be available in audio

That phrasing is so simple and clear. Would love to see it. It is a bit unfortunate that WCAG 3.0 is realistically at least 4 or 5 years away, but I do recognize that consensus takes time.

**Comment by nigelmegitt** [comment]
Strongly agree that the normative requirement needs to match the Understanding section and clearly state that the goal is that the content is understandable to people unable to see the video image, _not_ to add audio description. I've made proposals on #3443.

It's also not true that audio description cannot be mixed in the client player; finally we need to acknowledge that the main soundtrack volume can be lowered prior to mixing in audio description to make the description audio more understandable, if needed. This is common practice.

**Comment by mraccess77** [comment]
When the audio soundtrack is lowered to add audio description it should only be done when decorative audio is playing as lowering the volume can prevent the user from hearing important audible details including narration.  I've heard some video where the audio description obscured important details - some folks are hard of hearing and low vision and reducing volume with two audio channels at once can become inaudible.

**Comment by nigelmegitt** [comment]
@mraccess77 I agree that production quality is important in the way you describe - do you want to propose some text to make that clear?

Re multiple barriers to access, in this case poor hearing and poor vision, there's also the possibility of exposing a text alternative to the audio description, so that it can be presented using AT, for example on a Braille display. I've assumed we don't need to explicitly state that in 1.2.3 or 1.2.5 though.

**Comment by mbgower** [comment]
@gmjellin I believe your original issue has been sufficiently addressed by a number of changes that have been done to Audio Descriptions by the WCAG 2 Task Force in the past few months, including:
1. Creating a new technique [G226: Providing audio descriptions by incorporating narration in the soundtrack](https://www.w3.org/WAI/WCAG21/Techniques/general/G226)
2. Revising the language of the first note so that it now reads "For 1.2.3, 1.2.5, and 1.2.7, if all of the important information in the video track is already conveyed in the audio track, no additional audio description is necessary."

Please see #4323, #4324, #4327 and #4390 for some of the specifics on these and other updates to the Time-Based Media SCs that have been done.

We also have an open issue #4392. If you feel there is some part of your issue that is not sufficiently addressed by these changes, please reopen the issue and comment.

**Comment by nigelmegitt** [comment]
I believe the Understanding text is okay, but the normative text in 1.2.3 is still highly problematic: it no longer matches the intent and the improved text in Understanding. I think that would be a separate issue though, which I'm happy to open, if my search for an existing issue fails to find any.
